libraries/AP_HAL_ChibiOS/hwdef/scripts/convert_betaflight_unified.py

```diff
@@ -239,7 +239,7 @@ def convert_file(fname, board_id):
     f.write('''\n# Beeper
 %s BUZZER OUTPUT GPIO(80) LOW
 define HAL_BUZZER_PIN 80
-define HAL_BUZZER_ON 1
+define DEFAULT_BUZZ_ON_LVL 1
```

Comment:
I think we want to remove this - I'd bet a lot of the ones you're removing came in this way.

Reply:
yep